### PR TITLE
refactor: pack fontawesome instead of fetching via html

### PR DESCRIPTION
**Pull request by tynes**
- [x] pack fontawesome instead of fetching via htm
- [x] remove `node-sass` and related dependencies
- [x] remove bootstrap sass, use css instead

See https://github.com/bpanel-org/bpanel/issues/47

### CODE REVIEW

**Comment by bucko13** [comment]
We can probably also get rid of these two other deps now too right:
```
    "sass-lint": "^1.12.1",
    "sass-loader": "^6.0.6",
```

**Comment by tynes** [comment]
@bucko13 ready for you to take another look

File: webapp/containers/App/App.js [hunk 1]
```diff
@@ -123,7 +125,4 @@ const mapDispatchToProps = dispatch => {
   );
 };
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(App, 'App');
+export default connect(mapStateToProps, mapDispatchToProps)(App, 'App');
```
Comment (bucko13) [inline webapp/containers/App/App.js]: I think the linter wanted this split up. Did you run prettier on this file and it put it back?

**Comment by tynes** [comment]
Fixed the linting

**Comment by bucko13** [comment]
LGTM!

**Pull request merged**